mtl/test/mtl2milp_test.py

```diff
@@ -3,18 +3,14 @@
  See license.txt file for license information.
  @author: Gustavo A. Cardona, Cristian-Ioan Vasile
 """
-from antlr4 import InputStream, CommonTokenStream
-
 import sys
 sys.path.append('..')
 
-from mtl import Operation,  MTLFormula
+from antlr4 import InputStream, CommonTokenStream
+
```

Comment:
Remove these imports and use the `to_ast()` function instead. Also updated the code below: lines 18-22.